packages/base/src/UI5Element.js

```diff
@@ -101,6 +101,9 @@ class UI5Element extends HTMLElement {
 	 */
 	async connectedCallback() {
 		this.setAttribute(this.constructor.getMetadata().getPureTag(), "");
+		if (this.constructor.getMetadata().metadata.fastNavigation) {
```

Comment:
the `metadata` object is private, please create a getter for it, for example it would  look like this:
```js
this.constructor.getMetadata().supportsFastNavigation()
```